README.md

```diff
@@ -61,9 +61,9 @@ See [Deploying Hosts using Application Centric Deployment](https://docs.theforem
 
 See the [installation](https://theforeman.org/plugins/#2.Installation) chapter of the Foreman plugins documentation on how to install Foreman plugins.
 
-### TL;DR: 
+### TL;DR:
 
-    yum install tfm-rubygem-foreman_acd
+    dnf install rubygem-foreman_acd
```

Comment:
You can use foreman-installer: https://docs.theforeman.org/nightly/Deploying_Hosts_AppCentric/index-katello.html#Installing_ACD_on_Server_application_centric_deployment

If you don't mind, I suggest to drop this section in favor of a link to Foreman docs.

Reply:
Actually, I like that suggestion very much. Applied.